### PR TITLE
Fixed issue #17762: vulnerable verson of jQuery included in qtip2

### PR DESCRIPTION
**Pull request by Shnoulle**
Dev: remove unusued qtip2
Dev: included in third_party, but never registered

### CODE REVIEW

**Review by gabrieljenik — APPROVED** [review]
Seems ok. I added @olleharstedt 
Extra  pair of eyes seems beetter.

**Comment by Shnoulle** [comment]
Yep , qtip2 is use before, i think it's @olleharstedt that move to own system.

File: application/controllers/admin/translate.php [hunk 1]
```diff
@@ -320,72 +320,6 @@ private function showTranslateAdminmenu($iSurveyID, $survey_title, $tolang)
         return $this->_getLanguageList($iSurveyID, $tolang);
     }
 
-    /*
-    * _getSurveyButton() returns test / execute survey button
-    * @param string $iSurveyID Survey id
-    * @param string $menuitem_url Menu item url
-    */
-    private function _getSurveyButton($iSurveyID, $menuitem_url)
```
Comment (olleharstedt) [inline application/controllers/admin/translate.php:328]: This function is unused, or?

Comment (Shnoulle) [reply application/controllers/admin/translate.php:328]: Private function is easy to check if unused : it's unused :)

**Pull request merged**